eb_hooks.py

```diff
@@ -81,8 +81,11 @@ def post_ready_hook(self, *args, **kwargs):
     """
     # 'parallel' easyconfig parameter is set via EasyBlock.set_parallel in ready step based on available cores.
     # here we reduce parallellism to only use half of that for selected software,
-    # to avoid failing builds/tests due to out-of-memory problems
-    if self.name in ['TensorFlow', 'libxc']:
+    # to avoid failing builds/tests due to out-of-memory problems;
+    memory_hungry_build = self.name in ['libxc', 'TensorFlow']
+    # on A64FX systems, (HBM) memory is typically scarce, so we need to use fewer cores for some builds
+    memory_hungry_build_a64fx = self.name in ['Qt5']
```

Comment:
Shouldn't this also check for the CPU target, as it now seems to apply this for all CPUs?

Reply:
Yes, of course... Fixed in b07445e